### PR TITLE
chore: issue forms

### PR DESCRIPTION
**Pull request by almostSouji**
**Please describe the changes this PR makes and why it should be merged:**

- ✏️ draft while discord thread is still unresolved
- Change dev branch format to hash (since the current one embeds as mailto link)
- Not selecting anything prints as "No Answer" which is suboptimal for partials. instead, require partials and add a "No Partials" option (The more straightforward None/NONE is apparently a reserved keyword and cannot be used)

<details>
<summary>Images</summary>

image
image

</details>

**Status and versioning classification:**

- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -124,8 +127,8 @@ body:
     id: dev-release
     attributes:
       label: I have tested this issue on a development release
-      placeholder: discord.js@xx.x.x-dev.1530234593.d23280c
+      placeholder: d23280c
```
Comment (ImRodry) [inline .github/ISSUE_TEMPLATE/bug_report.yml:130]: I believe this should say the entire version number to make it easier to install the specified version when testing
```suggestion
      placeholder: xx.x.x-dev.1530234593.d23280c
```

Comment (almostSouji) [reply .github/ISSUE_TEMPLATE/bug_report.yml:130]: no

Comment (ImRodry) [reply .github/ISSUE_TEMPLATE/bug_report.yml:130]: Why 😐

Comment (almostSouji) [reply .github/ISSUE_TEMPLATE/bug_report.yml:130]: this inhibits embedding of the commit preview + short link, as mentioned in the thread. Parallel discussions in multiple places are not efficient or pleasant.

**Review by iCrawl — APPROVED** [review]

**Pull request merged**